README.md

```diff
@@ -1,3 +1,3 @@
 # Neutron Reflectivity Reference Data Sets
 
-This repository contains a Neutron Reflectivity data set taken on the [OFFSPEC](https://www.isis.stfc.ac.uk/Pages/offspec.aspx) instrument at the [ISIS Neutron and Muon Source](https://www.isis.stfc.ac.uk/) (UK). The [OFFSPEC data set](https://github.com/DAaaS-reference-data/Reflectivity/tree/main/OFFSPEC_runs_46992-4) represents a published study of the determination of a thin-film structure using Neutron Reflectivity: *ACS Appl. Mater. Interfaces* 2019, 11, 14, 13803–13811 (https://doi.org/10.1021/acsami.9b02700). The purpose of this data sets is to act as reference data set to test and validate Data Analysis as a Service (DAaaS) platforms, for example as part of the [EU ExPaNDS project](https://expands.eu/).
\ No newline at end of file
+This repository contains a Neutron Reflectivity data set taken on the [OFFSPEC](https://www.isis.stfc.ac.uk/Pages/offspec.aspx) instrument at the [ISIS Neutron and Muon Source](https://www.isis.stfc.ac.uk/) (UK). The OFFSPEC data set represents a published study of the determination of a thin-film structure using Neutron Reflectivity: *ACS Appl. Mater. Interfaces* 2019, 11, 14, 13803–13811 (https://doi.org/10.1021/acsami.9b02700). The purpose of this data sets is to act as reference data set to test and validate Data Analysis as a Service (DAaaS) platforms, for example as part of the [EU ExPaNDS project](https://expands.eu/).
```

Comment:
I would like to see the link to the OFFSPEC data set be kept as we might add more Reflectivity data sets in the future.